### ISSUE TITLE
fix missing/incorrect info in a11y help edits dialog

### ISSUE THREAD

**Issue by meganrogge** [microsoft/vscode#239374]
In the editor a11y dialog and in the edits one specifically, we need to fix the keybinding for accept `chatEditor.action.acceptHunk` and add `chatEditor.action.undoHunk`
